src/scripts/scrumHelper.js

```diff
@@ -341,118 +316,70 @@ function allIncluded() {
 		}
 		writeScrumBody();
 	}
+
+	// write github commits
+	function writeGithubCommits() {
+		if (!Array.isArray(githubPrCommitsData)) {
+			console.error('Invalid githubPrCommitsData: expected an array', githubPrCommitsData);
+			return;
+		}
+		const item = githubPrCommitsData;
+		for (let i = 0; i < item.length; i++) {
+			const html_url = item[i].html_url;
+			const urlParts = html_url.split('/');
+			const project = urlParts[4];
+			const commit = item[i].commit;
+			const commitMessage = commit.message;
+			const commitHash = item[i].sha;
+			const prMatch = commitMessage.match(/\(#(\d+)\)/);
+			let PRNumber;
```

Comment:
Please follow the same style of variable name.

Look like other parts are using camelCase, please follow it.